search/includes/classes/class-search.php

```diff
@@ -91,6 +91,13 @@ protected function setup_hooks() {
 
 		// Allow querying while a bulk index is running
 		add_filter( 'ep_enable_query_integration_during_indexing', '__return_true' );
+
+		// Set facet taxonomies size. Shouldn't currently be used, but it makes sense to have it set to a sensible
+		// default just in case it ends up in use so that the application doesn't error
+		add_filter( 'ep_facet_taxonomies_size', array( $this, 'filter__ep_facet_taxonomies_size' ), PHP_INT_MAX );
```

Comment:
Hmmm.. strange. I can't find any reference to `ep_facet_taxonomies_size` in their code or in their docs. Are you sure this is in use?

Reply:
NVM. Just saw your PR on our ElasticPress fork

Reply:
We don't have to set a priority on this filter - it's ok if it's overridden, it just can't be 10k by default.